terraform/postgres/database-dump/main.tf

```diff
@@ -46,7 +46,7 @@ data "aws_iam_policy_document" "assume_ecs_task_role" {
   dynamic "statement" {
     for_each = local.pipeline_tasks
     content {
-      sid    = "AllowPipelineAssumeRole"
+      sid    = "AllowPipelineAssumeRole${title(statement.value.to)}"
```

Comment:
I'd challenge whether this `sid` adds any value _anyway_ given the identifier is pretty clear already.

Reply:
Agreed, though I don't mind the convention if I'm honest.